pkg/core/consensus/round.go

```diff
@@ -257,12 +254,10 @@ func (c *Coordinator) CollectRoundUpdate(m bytes.Buffer) error {
 	c.stopped = false
 	go c.flushRoundQueue()
 
-	// TODO: the Coordinator should not send events. someone else should kickstart the
+	// TODO: the Coordinator should not send events. Someone else should kickstart the
 	// consensus loop
-	c.store.Dispatch(TopicEvent{
-		Topic: topics.Generation,
-		Event: Event{},
-	})
+	// TODO: interface - check this
+	c.store.Dispatch(message.New(topics.Generation, nil))
```

Comment:
The nil payload causes a panic at line 123, when a typecast is attempted to `consensus.InternalPacket`.

Reply:
Ops. Will change